tests/nlu/extractors/test_crf_entity_extractor.py

```diff
@@ -1,47 +1,60 @@
-from pathlib import Path
+import copy
 
 from typing import Dict, Text, List, Any
 
 import pytest
 
-from rasa.nlu.components import ComponentBuilder
-import rasa.nlu.train
+from rasa.engine.graph import ExecutionContext
+from rasa.engine.storage.resource import Resource
+from rasa.engine.storage.storage import ModelStorage
+from rasa.nlu import registry
 from rasa.nlu.config import RasaNLUModelConfig
-from rasa.nlu.model import Interpreter
 from rasa.nlu.featurizers.dense_featurizer.spacy_featurizer import SpacyFeaturizer
 from rasa.nlu.tokenizers.spacy_tokenizer import SpacyTokenizer
 from rasa.nlu.constants import SPACY_DOCS
+from rasa.nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+from rasa.shared.importers.rasa import RasaFileImporter
 from rasa.shared.nlu.constants import TEXT, ENTITIES
 from rasa.shared.nlu.training_data.message import Message
-from rasa.nlu.extractors.crf_entity_extractor import CRFEntityExtractor
-
-
-def pipeline_from_components(*components: Text) -> List[Dict[Text, Text]]:
-    return [{"name": c} for c in components]
+from rasa.nlu.extractors.crf_entity_extractor import CRFEntityExtractorGraphComponent
 
 
 async def test_train_persist_load_with_composite_entities(
-    component_builder: ComponentBuilder, tmp_path: Path
+    default_model_storage: ModelStorage, default_execution_context: ExecutionContext,
 ):
-    pipeline = [{"name": "WhitespaceTokenizer"}, {"name": "CRFEntityExtractor"}]
+    importer = RasaFileImporter(
+        training_data_paths=["data/test/demo-rasa-composite-entities.yml"]
+    )
+    training_data = importer.get_nlu_data()
 
-    _config = RasaNLUModelConfig({"pipeline": pipeline, "language": "en"})
+    tokenizer = WhitespaceTokenizer()
+    tokenizer.train(training_data)
 
-    (trainer, trained, persisted_path) = await rasa.nlu.train.train(
-        _config,
-        path=str(tmp_path),
-        data="data/test/demo-rasa-composite-entities.yml",
-        component_builder=component_builder,
+    crf_extractor = CRFEntityExtractorGraphComponent.create(
```

Comment:
optional: You could a fixture like I did [here](https://github.com/RasaHQ/rasa/blob/fc27727128b108c871394e375e219a7e27fabd35/tests/nlu/tokenizers/test_mitie_tokenizer.py#L17) to shorten this